src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -151,30 +140,43 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
     [amountAvailableCryptoPrecision, assetMarketData.price],
   )
 
-  const getOutboundFeeCryptoBaseUnit = useCallback(async (): Promise<Result<
-    string,
-    string
-  > | null> => {
-    if (!accountId) return null
+  const getOutboundFeeCryptoBaseUnit = useCallback(
+    async (
+      _quote?: Result<ThorchainSaversWithdrawQuoteResponseSuccess, string>,
+    ): Promise<Result<string, string> | null> => {
+      if (!accountId) return null
+
+      const maybeQuote = await (async () => {
+        if (_quote && _quote.isOk()) return _quote
+
+        // Attempt getting a quote with 100000 bps, i.e 100% withdraw
+        // - If this succeeds, this allows us to know the oubtound fee, which is always the same regarding of the withdraw bps
+        // and will allow us to gracefully handle amounts that are lower than the outbound fee
+        // - If this fails, we know that the withdraw amount is too low anyway, regarding of how many bps are withdrawn
+        setQuoteLoading(true)
+        const quote = await getThorchainSaversWithdrawQuote({ asset, accountId, bps: '10000' })
+        setQuoteLoading(false)
+        return quote
+      })()
+
+      // Neither the passed quote, nor the safer 10,000 bps quote succeeded
+      // Meaning the amount being withdraw *is* too small
+      if (maybeQuote.isErr()) {
+        console.error(maybeQuote.unwrapErr())
+        return Err(translate('trade.errors.amountTooSmallUnknownMinimum'))
```

Comment:
I see we dont ever actually use the monad, and are using monads as an implaicit try/catch in all cases, for example:
* as above where we return a totally different error monad (so monad returned isnt used), so try/catch can be used
* in 1 case, discard the error monad, in which case try/catch can be used
* in 6 cases, throw the error monad which attaches the wrong stack trace (it wont show you where the origingal error was created) `if (maybeQuote.isErr()) throw Error(maybeQuote.unwrapErr())`. If we used try/catch with vanilla error, the correct stack trace would be attached and isErr+unwrap+rethrow would not be required

I think for this use case we'd be better off using vanilla error handling unless we need to use the value inside the error monad for user-facing functionality, because it's going to give us more accurate errors with less double-handling of them